### DISCUSSION TITLE
Pygame running speed of website(pygbag) is much slower than Local-EXE

### DISCUSSION THREAD

**Discussion by songyuyx** [pygame-web/pygbag#135] [Q&A]
I made a simple games by pygame, and it is running good in Local-Exe files.

after use Pygbag to build it to a web-games,  the running speed is running very very slow in localhost:8000,  and also in itch.io. 

May I have any comments to fix the issue, thank you very much

**Comment by pmp-p** [comment]
usually the first thing to check is your display size : 1024x600 is ok enough including mobile.

incremental display also saves a lot of processing ( ie pygame.display.update() instead of flip() )

next to check is game logic, especially assets/surface handling.
